Polyhedron/demo/Polyhedron/Plugins/PMP/Engrave_text_plugin.cpp

```diff
@@ -514,7 +514,7 @@ public Q_SLOTS:
       if (A.x()<0) A=-A;
           angle = std::acos(A.x()/CGAL::sqrt(A.squared_length()));
           if ( A.y()<0 ) angle+=3*CGAL_PI/2.;
-          if (angle>2*CGAL_PI) angle-=CGAL_PI;
+          if (angle>2*CGAL_PI) angle-=2*CGAL_PI;
```

Comment:
Note that you can use `atan2` instead of `acos`. It returns directly an angle in (-π , +π], taking the sign of `y` into account.